src/Slider.jsx

```diff
@@ -29,7 +29,7 @@ function killEvent(ev) {
 
 class Button extends React.Component {
   render() {
-    return <button {...this.props} type="button" />;
+    return <button {...this.props} aria-label="Slider Handle" type="button" />;
```

Comment:
I started thinking about this a little more and think the better solution might be to change the API to instead require children, where the children are handles. 

You could imagine it looking something like this:
```javascript
<Rheostat
    min={1}
    max={100}
    values={[1, 100]}
>
  <Handle
    value={1}
    aria-label="Minimum price"
  />
  <Handle
    value={100}
    aria-label="Maximum price"
  />
</Rheostat>
```

For `Handle`, we'd need to add the proper roles and aria attributes in order to have it function as expected: https://www.w3.org/TR/wai-aria-practices-1.1/#slidertwothumb